src/components/utility/Typography/Typography.js

```diff
@@ -1,29 +1,47 @@
-import React from 'react';
+import React from "react";
+import styles from "./Typography.module.css";
+import classNames from "classnames";
+import PropTypes from "prop-types";
 
 export const TypographyElements = {
-  "body1": "p",
-  "body2": "p",
-  "span": "span",
-  "h1Small": "h1",
-  "h1Large": "h1",
-  "h2": "h2",
-  "h3": "h3",
-}
+  body1: "p",
+  body2: "p",
+  span: "span",
+  h1Large: "h1",
+  h1Small: "h1",
+  h2: "h2",
+  h3: "h3",
+  error: "p",
+};
 
-export default function Typography(
-  { elType, className = "", id = "", children }
-) {
+const Typography = ({ elType, className = "", id = "", children }) => {
   if (!Object.keys(TypographyElements).includes(elType)) {
-    throw TypeError("Unsupported element type for Typography component")
-  };
-
-  className = `${elType} ${className}`;
-
+    throw TypeError("Unsupported element type for Typography component");
+  }
   const Tag = TypographyElements[elType];
 
+  const combinedClassName = classNames(className, {
```

Comment:
Can you edit the PR description to explain what the `classNames` library does and why we need it? I can't tell from looking at this what it does.